platforms/springboot/src/main/java/io/hawt/springboot/HawtioPlugin.java

```diff
@@ -26,12 +27,31 @@ public class HawtioPlugin {
     private Boolean bustRemoteEntryCache;
     private String pluginEntry;
 
+    /**
+     *
```

Comment:
If you want to add javadoc, please also add the comment body from the first line.